dynamiqs/utils/utils.py

```diff
@@ -539,9 +538,13 @@ def ket_overlap(x: Tensor, y: Tensor) -> Tensor:
 def fidelity(x: Tensor, y: Tensor) -> Tensor:
     r"""Returns the fidelity of two states, kets or density matrix.
 
```

Comment:
Can you add the fidelity formulas for ket/ket and dm/dm cases? Can be inline math so it's not too heavy to look at.